### PR TITLE
Native Text component & Text Layout Manager to handle computation and drawing of paragraph content

### PR DESCRIPTION
**Pull request by pprabhu-ng**
Add native text layout manager module and support to use skia's skparagraph module.
Add support to measure and draw attributed string in text layout manager for paragraph content.
Add paragraph test app.

### CODE REVIEW

File: skia/BUILD.gn [hunk 1]
```diff
@@ -699,6 +700,21 @@ skia_source_set("skia_core_and_effects") {
   visibility = [ ":skia" ]
 }
 
+if (skia_support_skparagraph) {
```
Comment (hps1978) [inline skia/BUILD.gn]: Is this check required? 
skparagraph will be required as default, right?

File: skia/BUILD.gn [hunk 2]
```diff
@@ -36,6 +36,7 @@ skia_support_gpu = !is_ios
 skia_support_pdf = false
 skia_support_skottie = false
 enable_skia_wuffs_gif = true
+skia_support_skparagraph = true
```
Comment (hps1978) [inline skia/BUILD.gn]: skparagraph should be a default requirement for Text Component. Do we need to add this flag?

File: packages/react-native-skia/components/ParagraphTestApp.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { View, AppRegistry, Image, Text } from 'react-native';
```
Comment (hps1978) [inline packages/react-native-skia/components/ParagraphTestApp.js]: Image can be removed if not used in test code.

File: skia/BUILD.gn (hunk 1 above)
Comment (pprabhu-ng) [reply skia/BUILD.gn]: Since skparagraph is enabled by default, removed the check.

File: skia/BUILD.gn (hunk 2 above)
Comment (pprabhu-ng) [reply skia/BUILD.gn]: As skparagraph is required to enabled by default, removed this flag

File: packages/react-native-skia/components/ParagraphTestApp.js (hunk 3 above)
Comment (pprabhu-ng) [reply packages/react-native-skia/components/ParagraphTestApp.js]: As Image component is not used in the test app , removed it.

File: packages/react-native-skia/components/text/ParagraphTestApp.js [hunk 4]
```diff
@@ -10,32 +10,31 @@ const SimpleViewApp = React.Node = () => {
                justifyContent: 'center',
                alignItems: 'center',
                backgroundColor: 'white' }}>
-      <Text style={{ marginTop:'2' }}>
+      <Text style={{ marginTop:'2' ,marginLeft:'50',marginRight:'50'}}>
```
Comment (hps1978) [inline packages/react-native-skia/components/text/ParagraphTestApp.js]: layout props with absolute values should be numbers

**Review by hps1978 — APPROVED** [review]

**Review by hps1978 — APPROVED** [review]

**Pull request merged**